src/rebuild/srv.c

```diff
@@ -1376,7 +1375,13 @@ rebuild_ults(void *arg)
 			 * wait to start the next operation until the current
 			 * one completes
 			 */
-			if (pool_is_rebuilding(task->dst_pool_uuid))
+			uint64_t cur_ts = 0;
+
+			rc = daos_gettime_coarse(&cur_ts);
+			D_ASSERT(rc == 0);
+
+			if (cur_ts < task->dst_schedule_time ||
```

Comment:
Could you explain a bit the purpose of this delay and what's the problem of original dss_sleep(1000)?

Reply:
oh, the original implementation is blocking the current rebuild finish, see those following part after dss_sleep() in rebuild_task_ult().  So I move the wait into the rebuild task queue.